rotary/rules/polish.smk

```diff
@@ -135,10 +150,26 @@ rule polish_polca:
         mem=config.get("memory")
     shell:
         """
+        printf "\n\n### Replace bwa with bwa-mem2 ###\n" >> {log}
+        bwa_path=$(which bwa)
```

Comment:
I know that only some parts of our current snakemake code are robust to whitespaces, but I think it might make sense to make this bwa -> bwa-mem2 code whitespace robust. `$(which bwa)` would become `"$(which bwa)"`, and so on. If you'd like, I'd be happy to push a commit that implements this -- let me know!

Reply:
> I know that only some parts of our current snakemake code are robust to whitespaces, but I think it might make sense to make this bwa -> bwa-mem2 code whitespace robust. `$(which bwa)` would become `"$(which bwa)"`, and so on. If you'd like, I'd be happy to push a commit that implements this -- let me know!

@jmtsuji Feel free to fix this!

Reply:
@jmtsuji Feel free to make any changes necessary to update this branch.

Reply:
Just made some updates and pushed them into this PR. I haven't tested the updated rule but ran some basic bash syntax checks in command line to confirm that the general types of edits I made were OK. Hope this helps!